platform/install/quick-start-guide.mdx

```diff
@@ -2,67 +2,109 @@
+
+## Download and Install vCluster CLI
 
 Use one of the following commands to download the vCluster CLI binary from GitHub:
 
 <InstallCLI />
 
-## 2. Deploy the vCluster Platform
-
-:::caution Prerequisites
-Please make sure you meet the following requirements
-before running this command:
-1. Current kube-context has admin access to the cluster
-2. Helm v3 must be installed
-3. kubectl must be installed
+:::tip
+After installation, verify the successful installation of the vCluster CLI by running `vcluster version` in your terminal. This command should display the installed version of the vCluster CLI.
 :::
 
-Confirm that you are using the correct host cluster kube-context via `kubectl config current-context` and then run:
+## Deploy the platform
+
+Before deploying the platform, ensure you are using the correct host cluster kube-context. A host cluster kube-context is the configuration that determines which Kubernetes cluster and namespace `kubectl` commands will interact with. To confirm your current context, run:
+
+```bash title="Get the current Kubernetes context."
+kubectl config current-context
+```
 
-```bash
+Once you've confirmed the correct context, deploy the platform by running:
+
+```bash title="Deploy the platform."
 vcluster platform start
 ```
-This will deploy vCluster Platform into the host `vcluster-platform` namespace.
+
+This will deploy the platform onto the host-cluster in the `vcluster-platform` namespace.
 The UI will automatically open in your browser and log you in. You will be asked for your user details to create the admin user.
 
+:::info
+The deployment process typically takes about 5-10 minutes, depending on your cluster's resources and network speed.
+:::
+
+You should see output similar to the following:
+
+```bash title="Platform installation output."
+...
+
+##########################   LOGIN   ############################
+
+Username: admin
+Password: 27177595-21bc-4ff9-9f3a-51e0f722408b  # Change via UI or via: vcluster platform reset password
+
+# highlight-start
+Login via UI:  https://hth45c8.loft.host
+Login via CLI: vcluster login https://hth45c8.loft.host
+# highlight-end
+
+#################################################################
+
+vCluster Platform was successfully installed and can now be reached at: https://hth45c8.loft.host
+Thanks for using vCluster Platform!
+11:38:33 done You are successfully logged into vCluster Platform!
+- Use `vcluster create` to create a new virtual cluster
+- Use `vcluster platform add vcluster` to add an existing virtual cluster to a vCluster platform instance
+```
+
+This command deploys the platform into the `vcluster-platform` namespace of your Kubernetes cluster. Upon successful deployment, the UI will automatically open in your default web browser. You will be prompted to create an admin user by providing your user details.
```

Comment:
This content of the automatic UI and admin user ceration is duplicated in two places. This is the second mention. 

Do we need it in both places?

Reply:
It's probably due to partial duplicating some of the text. Fix incoming.